fdbmonitor/fdbmonitor.cpp

```diff
@@ -508,10 +546,13 @@ struct Command {
 		const char* id_s = ssection.c_str() + strlen(section.c_str()) + 1;
 
 		for (auto i : keys) {
-			if (!strcmp(i.pItem, "command") || !strcmp(i.pItem, "restart-delay") ||
-			    !strcmp(i.pItem, "initial-restart-delay") || !strcmp(i.pItem, "restart-backoff") ||
-			    !strcmp(i.pItem, "restart-delay-reset-interval") || !strcmp(i.pItem, "disable-lifecycle-logging") ||
-			    !strcmp(i.pItem, "delete-envvars") || !strcmp(i.pItem, "kill-on-configuration-change")) {
+			if (!strcmp(i.pItem, "command") || !isEqualIgnoreHyphenAndUnderscore(i.pItem, "restart-delay") ||
+			    !isEqualIgnoreHyphenAndUnderscore(i.pItem, "initial-restart-delay") ||
```

Comment:
You could call this something like isParameterNameEqual() and use it for all parameters (even "command"). Then it could just contain any logic we want to use for equality.